rbi/stdlib/reline.rbi

```diff
@@ -0,0 +1,223 @@
+# typed: __STDLIB_INTERNAL
+
+module Reline
```

Comment:
Overall, would you mind committing an RBI file that has been generated by `srb rbi gems` or `tapioca`?

Doing so will automatically expand things like `Struct.new`, `def_single_delegator`, attr_reader`, etc. to normal method defs, which will make it easier for people to incrementally add or fix types in the future. For example, it's not easy right now to provide a type annotation for a single method of a class created with `Struct.new`.

Reply:
I think I did this correctly. Let me know if there's anything else I should do.